isotpsniffer.c

```diff
@@ -302,6 +315,17 @@ int main(int argc, char **argv)
 
 	setsockopt(t, SOL_CAN_ISOTP, CAN_ISOTP_OPTS, &opts, sizeof(opts));
 
+	if ((setsockopt(s, SOL_CAN_ISOTP, CAN_ISOTP_LL_OPTS, &llopts, sizeof(llopts))) < 0) {
+		perror("setsockopt");
+		close(s);
```

Comment:
is close(s) needed before exit?
AFAIK exit closes all file descriptors

Reply:
It does, but I learned in other projects that it is "good style" to close them explicitely.

Reply:
But then both sockets (s & t) need to be taken care of, right?

Reply:
If you want to have proper error handling, please add it in a separate patch. Please use goto-style error handling, if you have more than one resource.

Reply:
yep. better to fix the error handling properly before merging. will update this PR.